core/components/TaskForm/hooks/useSubmitForm.ts

```diff
@@ -46,7 +46,11 @@ export const useSubmitForm = ({ id, taskName, status }: TaskFormProps) => {
     dispatch(editTask(task) as any);
   };
 
-  const handlePressedEnter = (event: any, handleEditClick: Function) => {
+  const handlePressedEnter = (
+    event: React.SyntheticEvent,
+    handleEditClick: Function
```

Comment:
don't use `Function` add the `handleEditClick` function type

Reply:
Done